coordinator/src/routes/health/get-status.ts

```diff
@@ -30,5 +30,25 @@ export default [
         commitId: process.env.COMMIT_ID
       })
     }
+  },
+  {
+    method: "GET",
+    path: "/_validatormetrics/{path*}",
```

Comment:
`/_validatormetrics` seems a bit ugly, can we capitalise M maybe? or use a dash like in `/$process-message`?

Reply:
its not super clear what kind of values `path` would have? are there multiple endpoints on the validator that report metrics?

Reply:
It is a bit ugly. I dont think the name is too important, this is an internal api for diagnostics, and we might not even go to production with it

Reply:
`{path*}` just picks up any path components and proxies them onto the metrics endpoint